airflow/utils/log/file_task_handler.py

```diff
@@ -142,7 +142,7 @@ def read(self, task_instance, try_number=None):
         next_try = task_instance.try_number
 
         if try_number is None:
-            try_numbers = list(range(next_try))
+            try_numbers = list(range(1, next_try))
         elif try_number < 0:
```

Comment:
< 1?

Reply:
Previous this had `range(2)` (for example) meaning `try_number == [0,1]`, which we then added one to when we rendered it to the file.

Now with next_try having a value of 3 we want `[1,2]`, not `[0,1,2]`

Reply:
1 being the number of the first try, not 0 -- i.e. `.../1.log`

Reply:
Are we talking about the same thing here? The check

```
elif try_number < 0
```

Should in my opinion check `< 1` as log files with 0 dont exist?

Reply:
Oh! yes!. good point. i thought the comment was about my diff, even though it was targetting the next line.